tests/project/_common/test_movement.py

```diff
@@ -0,0 +1,539 @@
+    responses = _project_clean_up(client, new_project_name, new_unified_dataset_name)
+    print(responses)
+
+
+@pytest.mark.parametrize(
+    "project_to_export, existing_project_name, "
+    "include_additive_artifacts, include_destructive_artifacts",
+    [
+        (
+            CONFIG["projects"]["minimal_categorization"],
+            "minimal_incomplete_categorization",
+            None,
+            None,
+        ),
+        (CONFIG["projects"]["minimal_mastering"], "minimal_incomplete_mastering", None, None,),
+        (
+            CONFIG["projects"]["minimal_golden_records"],
+            "minimal_incomplete_golden_records",
+            None,
+            None,
+        ),
+        (
+            CONFIG["projects"]["minimal_schema_mapping"],
+            "minimal_incomplete_schema_mapping",
+            None,
+            None,
+        ),
+        (
+            CONFIG["projects"]["minimal_categorization"],
+            "minimal_incomplete_categorization",
+            [CategorizationArtifacts.CATEGORIZATION_VERIFIED_LABELS],
+            [
+                CategorizationArtifacts.CATEGORIZATION_MODEL,
+                CategorizationArtifacts.CATEGORIZATION_TAXONOMIES,
+            ],
+        ),
+        (
+            CONFIG["projects"]["minimal_mastering"],
+            "minimal_incomplete_mastering",
+            [MasteringArtifacts.RECORD_PAIR_VERIFIED_LABELS],
+            [MasteringArtifacts.CLUSTERING_MODEL, MasteringArtifacts.PUBLISHED_CLUSTERS],
+        ),
+        (
+            CONFIG["projects"]["minimal_golden_records"],
+            "minimal_incomplete_golden_records",
+            None,
+            [GoldenRecordsArtifacts.GR_RULES],
+        ),
+        (
+            CONFIG["projects"]["minimal_schema_mapping"],
+            "minimal_incomplete_schema_mapping",
+            [SchemaMappingArtifacts.UNIFIED_ATTRIBUTES],
+            [SchemaMappingArtifacts.TRANSFORMATIONS],
+        ),
+    ],
+)
+@mock_api()
+def test_import_existing(
+    project_to_export: str,
+    existing_project_name: str,
+    include_additive_artifacts: Optional[List[str]],
+    include_destructive_artifacts: Optional[List[str]],
+):
+    client = utils.client.create(**CONFIG["toolbox_test_instance"])
+    # project to export
+    project = client.projects.by_resource_id(project_to_export)
+    # export a project
+    op = categorization.export_artifacts(
+        project=project,
+        artifact_directory_path="/home/ubuntu/tamr/projectExports",
+        exclude_artifacts=None,
+        asynchronous=False,
+    )
+    assert op.succeeded()
+
+    # get artifact path
+    description = op.description
+    artifact_path = str(description).split(" ")[-1]
+
+    assert artifact_path[0] == "/"
+    assert os.path.splitext(artifact_path)[1] == ".zip"
+
+    # get existing project
+    existing_project = client.projects.by_name(existing_project_name)
+
+    # test import into existing project
+    op = categorization.import_artifacts(
```

Comment:
The `categorization` module is being used here for export/import of all project types. This exposes an unnecessary constraint on how this is being called i.e. the api is the same regardless of which module you import. The preferred behavior I believe would be not to require calling the correct project type (since it's not actually required to do this). But to make the mechanism for calling this common to all projects. For example: `tbox.projects.export_artifacts`

@pareantoine @skalish @tiems90 what do you think?

Reply:
Resolved with https://github.com/Datatamer/tamr-toolbox/pull/60